datanode/server.go

```diff
@@ -233,6 +240,20 @@ func doStart(server common.Server, cfg *config.Config) (err error) {
 	// start metrics (LackDpCount, etc.)
 	s.startMetrics()
 
+	// start cpu sampler
+	s.cpuSamplerDone = make(chan struct{})
+	go func() {
+		for {
+			select {
+			case <-s.cpuSamplerDone:
+				return
+			default:
+				used := loadutil.GetCpuUsedPercent(cpuSampleDuration)
```

Comment:
what is the interval of GetCpuUsedPercent

Reply:
`cpuSampleDuration`